generators/loopback/templates/server/package.json

```diff
@@ -3,15 +3,16 @@
   "version": "1.0.0",
   "main": "server/server.js",
   "scripts": {
-    "lint": "eslint .",
-    "migrate:create": "./node_modules/.bin/db-migrate create --sql-file",
-    "migrate:down": "./node_modules/.bin/db-migrate down -e $NODE_ENV",
-    "migrate:up": "./node_modules/.bin/db-migrate up -e $NODE_ENV",
+    "lint": "eslint ./server",
+    "migrate:create": "db-migrate create --sql-file",
+    "migrate:down": "db-migrate down -e $NODE_ENV",
+    "migrate:up": "db-migrate up -e $NODE_ENV",
     "posttest": "npm run lint && nsp check",
-    "start": " npm run migrate:up && yarn stop && ./node_modules/.bin/pm2 start pm2.yml",
-    "start:prod": "npm run migrate:up -- -e production && NODE_ENV=production ./node_modules/.bin/pm2 start pm2.yml",
-    "start:watch": "npm run migrate:up && ./node_modules/.bin/pm2 start server/server.js --watch ./server",
-    "stop": "./node_modules/.bin/pm2 kill"
+    "start": " npm run migrate:up && yarn stop && pm2 start pm2.yml",
+    "start:prod": "npm run migrate:up -- -e production && NODE_ENV=production pm2 start pm2.yml",
+    "start:watch": "npm run migrate:up && pm2 start server/server.js --watch ./server",
+    "stop": "pm2 kill",
```

Comment:
The `node_modules/.bin/` directory is automatically added to the path when writing npm scripts, so you don't need to prefix all commands by this path.